plan/plan.go

```diff
@@ -177,13 +180,6 @@ func targetChanged(desired, current *endpoint.Endpoint) bool {
 	return !desired.Targets.Same(current.Targets)
 }
 
-func shouldUpdateTTL(desired, current *endpoint.Endpoint) bool {
```

Comment:
What happens with other providers in that case? As I can see only Cloudflare has been made aware of this change.

Reply:
@njuettner , other provider will use this method, moved as is to BaseProvider struct - https://github.com/kubernetes-sigs/external-dns/pull/1822/files#diff-efafdf56d71f1a3f19d003c8bad9fbadb8ee34e873cfa41362ca880e9f7c5fd3R269-R277